### PR TITLE
Docker volume name

### LINKED ISSUE #321: Hold chain data in named docker volume
I just noticed this, we should maybe store data on separate volume and maybe add button to delete chain data.
Otherwise we won't be able to update beacon node without resyncing. 

This is quite annoying part of managing docker node.

cc @morrigan

_Originally posted by @mpetrunic in https://github.com/NodeFactoryIo/ChainGuardian/pull/318#discussion_r530918459_

### PR DESCRIPTION
**Pull request by BeroBurny**
closes #321

### CODE REVIEW

File: src/renderer/ducks/beacon/sagas.ts [hunk 1]
```diff
@@ -26,18 +26,32 @@ export function* pullDockerImage(
...
         switch (network) {
             default:
                 yield put(
-                    addBeacon(`http://localhost:${ports[1].local}`, {
-                        id: (yield call(BeaconChain.startBeaconChain, SupportedNetworks.LOCALHOST, ports)).getName(),
+                    addBeacon(`http://localhost:${rpcPort}`, {
+                        id: (yield call(BeaconChain.startBeaconChain, SupportedNetworks.LOCALHOST, {
+                            ports,
+                            // eslint-disable-next-line max-len
+                            cmd: `lighthouse beacon_node --network ${network} --port ${libp2pPort} --discovery-port ${discoveryPort} --http --http-address 0.0.0.0 --http-port ${rpcPort} --eth1-endpoints ${eth1Url}`,
```
Comment (mpetrunic) [inline src/renderer/ducks/beacon/sagas.ts:50]: can you separate cmd and docker imaeg chain data into some lighthouse configuration object. Maybe make it implement some interface.
```typescript
interface IBeconNodeConfiguration {
    defaultParams: IBeaconNodeParams;
    chainDataLocation: string;
    dockerImage: string;
    getCmd(params: IBeaconNodeParams): string;
}
```
If you don't think this belongs in this PR, please open issue for this.

File: src/renderer/services/eth2/networks/pyrmont.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+        ...minimalBeaconConfig,
+        params: {
+            ...minimalBeaconConfig.params,
+            GENESIS_FORK_VERSION: Buffer.from("0x00000001"),
+        },
+    },
+    eth1Provider: new ethers.providers.JsonRpcProvider("https://goerli.infura.io/v3/9d117da5b35f4503b142619ad690fc6a"),
+    dockerConfig: {
+        name: "lighthouse_validators",
+        image: "sigp/lighthouse",
```
Comment (mpetrunic) [inline src/renderer/services/eth2/networks/pyrmont.ts:26]: add version tag

File: src/renderer/services/eth2/networks/pyrmont.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    eth2Config: {
+        ...minimalBeaconConfig,
+        params: {
+            ...minimalBeaconConfig.params,
+            GENESIS_FORK_VERSION: Buffer.from("0x00000001"),
+        },
+    },
+    eth1Provider: new ethers.providers.JsonRpcProvider("https://goerli.infura.io/v3/9d117da5b35f4503b142619ad690fc6a"),
+    dockerConfig: {
+        name: "lighthouse_validators",
```
Comment (mpetrunic) [inline src/renderer/services/eth2/networks/pyrmont.ts:25]: we don't have lighthouse_validators and we will never have 😄

File: src/renderer/ducks/beacon/sagas.ts (hunk 1 above)
Comment (BeroBurny) [reply src/renderer/ducks/beacon/sagas.ts:50]: opened #347 better bother with this when the time comes

**Review by mpetrunic — APPROVED** [review]

**Pull request merged**